scalajs/src/main/scala/clue/js/WebSocketJSBackend.scala

```diff
@@ -84,6 +85,10 @@ final class WebSocketJSBackend[F[_]: Async: Logger](dispatcher: Dispatcher[F])
               } yield ()
             dispatcher.unsafeRunAndForget(close)
           }
+
+          Applicative[F].pure(
+            Sync[F].delay(ws.close(1001, "Web Socket initialization canceled by client")).some
```

Comment:
Browsers will only accept 1000 here.
https://github.com/http4s/http4s-dom/blob/ee5d9f1ea95659877ebdeb47b0decbe5b36d041f/dom/src/main/scala/org/http4s/dom/WebSocketClient.scala#L109-L111

Reply:
Oh ok, I was going by this: https://www.rfc-editor.org/rfc/rfc6455.html#section-7.4.1.

Browsers don't implement that?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/close#parameters

> The value must be an integer: either 1000, or else a custom code of your choosing in the range 3000-4999.